### ISSUE TITLE
Enable 2 factor module configuration via environment variables

### ISSUE THREAD

**Issue by aturki** [scheb/2fa#54]
Right now configuring modules via environment variables is not supported for instance, the following configuration is does not work:

```yaml
scheb_two_factor:
  email:
    enabled: '%env(ENABLE_2FA)%'
```

**Comment by Spomky** [comment]
What about the `bool` env processor ?
https://symfony.com/doc/current/configuration/env_var_processors.html#built-in-environment-variable-processors

```yaml
scheb_two_factor:
  email:
    enabled: '%env(bool:ENABLE_2FA)%'
```

**Comment by aturki** [comment]
That was my first approach indeed. But it seems that Symfony is delaying the evaluation of these variables until later in the container's construction process. 
Since we are actually executing code in the bundle's configuration it's not possible to use the processors [ref](https://symfony.com/doc/current/configuration/using_parameters_in_dic.html)

**Comment by scheb** [comment]
Ok, I just learned something new. Environment variables are only replaced after the container building phase and are represented by their placeholder string until then (which is in our case always evaluating to `true`).

When you need the environment variable value during container building, you have to take care of transforming the variables. `resolveEnvPlaceholders` (as suggested in your PR) seems to be the way to go: https://github.com/symfony/symfony/issues/21420#issuecomment-275459297

**Comment by scheb** [comment]
Feature released with version 5.5.0.
